packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -131,6 +131,21 @@ export async function execCommand(command: string, options: SpawnOptions = {}) {
   return true;
 }
 
+export function* traverseUpDirectories(start: string, root?: string) {
```

Comment:
Let's change this from two args to a single obj arg

```suggestion
export function* traverseUpDirectories(props: {start: string, root?: string}) {
```

This is much safer long term if we make changes to arguments

Reply:
29c0e0adb542b2ae793dee598a1aec9e6a1627db